storage/filesystem/dotgit/dotgit.go

```diff
@@ -278,6 +394,87 @@ func (d *DotGit) ForEachObjectHash(fun func(plumbing.Hash) error) error {
 	return nil
 }
 
+func (d *DotGit) cleanObjectList() {
+	d.objectMap = nil
+	d.objectList = nil
+}
+
+func (d *DotGit) genObjectList() error {
```

Comment:
This function is not covered by a test

Reply:
By default go coverage only annotates files from the same package tested. The test that exercise this part belongs to `storage/filesystem`. That's why it does not appear in coverage. The same happens with other functions. I'll add new tests in `storage/filesystem/dotgit` that forces the usage of those functions.